samples/chaincode/helloworld/README.md

```diff
@@ -307,7 +305,7 @@ make[1]: Leaving directory '/home/bcuser/work/src/github.com/hyperledger-labs/fa
 ## Time to test!
 Next step is to test the chaincode by invoking transactions, for which you need a basic Fabric network with a channel. You will use the FPC test framework to bring up a Fabric network in which the helloworld code can be executed as a chaincode _in an SGX enclave_.  The Fabric network contains just a single peer and orderer node as defined in `$FPC_PATH/integration/config/`.  Please note the FPC-specific additions to the `core.yaml` to the standard Fabric configurations.  These are marked as `FPC Addition`;  these enable the integration points with Fabric and have to be replicated if you want to use your own Fabric configuration.
 
-Create a file `test.sh` in `$FPC_PATH/examples/helloworld` folder as follows.  Note that the initial lines in the script points to files and folders in FPC framework.
+Create a file `test.sh` in `$FPC_PATH/samples/chaincode/helloworld` folder as follows.  Note that the initial lines in the script points to files and folders in FPC framework.
```

Comment:
why is the code for test.sh defined as Makefile?

Reply:
good catch